src/pages/index.js

```diff
@@ -292,6 +296,13 @@ const useStyles = makeStyles((theme) => ({
     marginLeft: theme.spacing(1),
     color: 'white',
   },
+  searchbarNotification: {
+    zIndex: 99999,
+    position: 'absolute',
+    width: theme.spacing(60),
+    top: theme.spacing(0),
+    left: theme.spacing(15),
```

Comment:
In the future, don't rely on `spacing` for positioning like this. IMO, it's primarily useful for consistency within components and layouts, but this is a case where we wouldn't want a position shift if we were to tweak the theme's default spacing.